rampwf/tests/test_kits.py

```diff
@@ -10,9 +11,29 @@
 PATH = os.path.dirname(__file__)
 
 
+def skip_windows_py27():
+    return pytest.mark.skipif(
+        (sys.platform == 'win32') and (sys.version_info < (3, 5)),
+        reason="tensorflow not available")
+
+
+def skip_no_tensorflow():
+    try:
+        import tensorflow
+    except ImportError:
+        return pytest.mark.skip(reason='tensorflow not available')
+    return pytest.mark.basic
+
+
 def _generate_grid_path_kits():
-    return [(os.path.abspath(path_kit,))
-            for path_kit in sorted(glob.glob(os.path.join(PATH, 'kits', '*')))]
+    grid = []
+    for path_kit in sorted(glob.glob(os.path.join(PATH, 'kits', '*'))):
+        if 'digits' in path_kit:
+            grid.append(pytest.param(os.path.abspath(path_kit),
+                                     marks=skip_no_tensorflow()))
+        else:
+            grid.append(os.path.abspath(path_kit))
+    return grid
```

Comment:
I personally actually preferred the manual list, but I should have said that in a previous PR .. :)